lib/main.js

```diff
@@ -875,6 +862,29 @@ so you can easily test run multiple events.
       console.log(err)
     })
   }
+
+  _loadAWSCredentials (program, options) {
+    if (typeof options !== 'object' || !options.region) {
+      options = {region: null}
+    }
+
+    const awsSecurity = { region: options.region }
+
+    if (program.profile) {
+      aws.config.credentials = new aws.SharedIniFileCredentials({
```

Comment:
It seems that there is a sense of unity if you make it to `awsSecurity.credentials = new aws.SharedIniFileCredentials(...) `, what do you think?